examples/run_CPMG.py

```diff
@@ -31,91 +31,71 @@
 # {{{set phase cycling
 phase_cycling = True
 if phase_cycling:
-    ph1_cyc = r_[0, 1, 2, 3]
-    nPhaseSteps = 4
+    Nph_overall = r_[0,1,2,3]
+    Nph_diff = r_[0,2]
+    ph1_cyc = array([(j+k)%4 for k in Nph_overall for j in Nph_diff])
+    ph2_cyc = array([(k+1)%4 for k in Nph_overall for j in Nph_diff])
+    nPhaseSteps = len(Nph_overall) * len(Nph_diff)
 if not phase_cycling:
     ph1_cyc = 0.0
+    ph2_cyc = 0.0
+    Nph_overall = r_[0]
+    Nph_diff = r_[0]
     nPhaseSteps = 1
```

Comment:
none of this should be here.  Running without phase cycling is not something we want to do.

Reply:
fixed